0800-appendix-198116-waterfall.Rmd

```diff
@@ -27,12 +27,10 @@ my_site3 <- 2022091003
 PSCIS crossing `r as.character(my_site)` is located on
 `r fpr::fpr_my_pscis_info()`, in the town of New Hazelton, BC. PSCIS
 crossing `r as.character(my_site)` is located on
-`r fpr_my_pscis_info(col_pull = road_name)`. Waterfall Creek does not
+`r fpr_my_pscis_info(col_pull = road_name)`, and is approximately 4km upstream from the confluence with Station Creek. Waterfall Creek does not
```

Comment:
Need to explain this.  (ie. because the stream is not mapped correctly in the freshwater atlas.... )

If we use field data and GIS to manually pull out habitat estimates I wouldn't call it "modelling"